aspect/intellij_info_impl.bzl

```diff
@@ -359,11 +365,13 @@ def collect_go_info(target, ctx, semantics, ide_info, ide_info_file, output_grou
             # if the .go file isn't in 'files', build the .a and .x files instead
             generated = genfiles
     else:
-        proto_sources = _collect_generated_proto_go_sources(target)
-        if not proto_sources:
+        generated_sources = _collect_generated_proto_go_sources(target)
+        if not generated_sources:
+            generated_sources = _collect_generated_go_sources(target)
+        if not generated_sources:
             return False
```

Comment:
This can stay the same, just need to change the variable name.

Reply:
Added a new diff to address this.